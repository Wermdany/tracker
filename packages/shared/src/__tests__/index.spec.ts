import * as modules from "..";

describe("test shared output modules", () => {
  it("should shared output modules", () => {
    expect(Object.keys(modules)).toMatchSnapshot();
  });
});
