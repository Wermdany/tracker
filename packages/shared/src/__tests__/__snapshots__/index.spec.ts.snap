// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test shared output modules should shared output modules 1`] = `
[
  "isType",
  "getType",
  "isFn",
  "isArr",
  "isPlainObj",
  "isStr",
  "isBool",
  "isNum",
  "isRegexp",
  "isUndefined",
  "isDate",
  "isObj",
  "isElement",
  "encrypt",
  "decrypt",
  "qs",
  "noop",
  "omit",
  "pick",
  "hasOwn",
  "transProfile",
  "includes",
  "excludes",
  "pickParse",
  "formatNumber",
  "getReferrer",
  "getTimezoneOffset",
  "getScreenWidth",
  "getScreenHeight",
  "getViewportWidth",
  "getViewportHeight",
  "getUrl",
  "getPathName",
  "getTitle",
  "getCurrentElementSelector",
  "getElementSelector",
  "getElementPath",
  "getElementClassName",
  "getElementContent",
  "getElementTagName",
  "getElementId",
  "getElementClassList",
  "getAttributeNames",
  "BaseStorage",
  "createLocalStorage",
  "createSessionStorage",
  "createCookie",
]
`;
